esx_circleprogressbar/nui/script.js

```diff
@@ -0,0 +1,7 @@
+document.addEventListener("DOMContentLoaded", function() {
```

Comment:
that's this line here, what is document and why are you using it? Personally, you should get this working and submit a new PR!